openpype/modules/sync_server/launch_hooks/pre_copy_last_published_workfile.py

```diff
@@ -119,6 +119,18 @@ def execute(self):
             "task": {"name": task_name, "type": task_type}
         }
 
+        # Add version filter
+        workfile_version = self.launch_context.data.get("workfile_version", -1)
```

Comment:
If we initialize it as `version_index = self.launch_context.data.get("workfile_version", -1)`

There's no need for lines `131` and `132`